### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by shemau**
### Description

Create public certificate module, initial version.

This is draft, pending the first release of terraform-ibm-secrets-manager-public-cert-engine module.

This is the first version, so the upgrade test will be skipped.
code owners are set from the template, but fixed in this PR. Future reviewers should be set appropriately.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [x] Major release (`X.x.x`)

##### Release notes content

Required version 1.0.0.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
 <!-- Update the title -->
 # Terraform Modules Template Project
```
Comment (ocofaigh) [inline README.md:2]: Update the readme title

**Comment by ocofaigh** [comment]
@alex-reiff could you take a look please?

**Review by alex-reiff — DISMISSED** [review]
looks good

File: README.md [hunk 2]
```diff
@@ -1,27 +1,25 @@
 <!-- Update the title -->
-# Terraform Modules Template Project
+# Secrets manager public cert module
 
-<!--
-Update status and "latest release" badges:
-  1. For the status options, see https://terraform-ibm-modules.github.io/documentation/#/badge-status
-  2. Update the "latest release" badge to point to the correct module's repo. Replace "terraform-ibm-module-template" in two places.
--->
 [![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
```
Comment (ocofaigh) [inline README.md:4]: Can you update this to the graduated status?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
2 minor comments from me. Can you also please update the `.github/settings.yml` and add repo name, description, and some topics. Ensure to add the `core-team`, `graduated` and `supported` topics.

File: README.md (hunk 2 above)
Comment (shemau) [reply README.md]: fixed in latest commit

**Comment by shemau** [comment]
> 2 minor comments from me. Can you also please update the `.github/settings.yml` and add repo name, description, and some topics. Ensure to add the `core-team`, `graduated` and `supported` topics.

Fixed in latest commit

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**